examples/homography_estimation.py

```diff
@@ -312,6 +318,7 @@ def run():
         err_fn=homography_error_fn,
         dim=1,
         aux_vars=[feat1, feat2],
+        autograd_functorch=True,
```

Comment:
Can you make this a parameter at the top of run()? In an future PR we can add back argparse / hydra to make it more command line friendly.